sdk/core/azure-core-http-netty/README.md

```diff
@@ -53,16 +53,16 @@ HttpClient client = new NettyAsyncHttpClientBuilder()
 
 ### Enabling Logging
 
-Azure SDKs for Java offer a consistent logging story to help aid in troubleshooting application errors and expedite
+Azure SDKs for Java provide a consistent logging story to help aid in troubleshooting application errors and expedite
 their resolution. The logs produced will capture the flow of an application before reaching the terminal state to help
 locate the root issue. View the [logging][logging] wiki for guidance about enabling logging.
 
 ## Next steps
 
 ## Contributing
 
-If you would like to become an active contributor to this project please follow the instructions provided in [Microsoft
-Azure Projects Contribution Guidelines](http://azure.github.io/guidelines.html).
+If you would like to become an active contributor to this project please follow the instructions provided in 
+[MicrosoftAzure Projects Contribution Guidelines](http://azure.github.io/guidelines.html).
```

Comment:
Should this have a space between the words?

Reply:
Yes, good catch.